client/client.go

```diff
@@ -324,7 +326,6 @@ func (m *K8sClient) WaitPodsReady(ns string, rcd *ReadyCheckData, expectedPodCou
 					return nil
 				}
 			}
-			time.Sleep(K8sStatePollInterval)
```

Comment:
I noticed this would block yielding to `timeout.C`, so I swapped it for a `time.Ticker`.